tests/acceptance/features/bootstrap/GraphContext.php

```diff
@@ -340,29 +400,69 @@ public function adminHasRetrievedGroupListUsingTheGraphApi(): array {
 	}
 
 	/**
-	 * returns a list of members in group
-	 *
-	 * @param string $group
 	 *
 	 * @return array
 	 * @throws Exception
 	 * @throws GuzzleException
 	 */
-	public function theAdminHasRetrievedMembersListOfGroupUsingTheGraphApi(string $group): array {
-		$response = GraphHelper::getMembersList(
+	public function adminHasRetrievedGroupListUsingTheGraphApi(): array {
```

Comment:
now, this may be empty if the request has not been made.
```suggestion
	public function adminHasRetrievedGroupListFromTheLastResponse(): array {
```

Reply:
The function doesn't get the groups list from the last response rather it sends a new request